pkg/collect/host_tls.go

```diff
@@ -35,40 +35,43 @@ func (c *CollectHostTLS) Collect(progressChan chan<- interface{}) (map[string][]
 
 	conn, err := tls.Dial("tcp", c.hostCollector.Address, conf)
 	if err != nil {
-		log.Println("Error in Dial", err)
-		return nil, errors.Wrap(err, "failed to dial tls")
-	}
-	defer conn.Close()
-	certs := conn.ConnectionState().PeerCertificates
-	cleanedCerts := make([]types.CertInfo, len(certs))
-	for i, cert := range certs {
-		cleanedCerts[i] = types.CertInfo{
-			Issuer:    cert.Issuer.CommonName,
-			Subject:   cert.Subject.CommonName,
-			Serial:    cert.SerialNumber.String(),
-			NotBefore: cert.NotBefore.Format(time.RFC3339),
-			NotAfter:  cert.NotAfter.Format(time.RFC3339),
-			IsCA:      cert.IsCA,
-			Raw:       cert.Raw,
+		tlsInfo.Error = err.Error()
```

Comment:
I guess I had imagined you would piggyback on this

https://github.com/replicatedhq/troubleshoot/blob/cbb22e27a81246a39a705adfe4189ade848e7bdf/pkg/collect/host_network.go#L73